lib/config.ts

```diff
@@ -1 +1 @@
-export const enableLogs = false; // change this to enable debug logs
+export const enableLogs = true; // change this to enable debug logs
```

Comment:
please don't forget to change this back before merging

Reply:
see comment below: https://github.com/duckduckgo/autoconsent/pull/22#issuecomment-1204897857